.devcontainer/Dockerfile

```diff
@@ -336,17 +336,30 @@ COPY lte/gateway/deploy/roles/magma/files/patches/aioeventlet_fd_exception.patch
 RUN patch -N -s -f /usr/local/lib/python3.8/dist-packages/aioeventlet.py < /tmp/aioeventlet_fd_exception.patch && \
     rm -rf /tmp/*
 
+COPY . ${MAGMA_ROOT}
+
 ### install eggs (lte, orc8r)
-COPY /lte/gateway/python/ ${MAGMA_ROOT}/lte/gateway/python/
 WORKDIR ${MAGMA_ROOT}/lte/gateway/python/
 RUN pip3 install --quiet --upgrade --no-build-isolation --no-cache-dir --verbose --editable .[dev] && \
     rm -rf lte.egg-info
 
-COPY /orc8r/gateway/python/ ${MAGMA_ROOT}/orc8r/gateway/python/
 WORKDIR ${MAGMA_ROOT}/orc8r/gateway/python/
 RUN pip3 install --quiet --upgrade --no-build-isolation --no-cache-dir --verbose --editable .[dev] && \
     rm -rf orc8r.egg-info
 
+#### protos
+ARG PYTHON_LIB=/usr/local/lib/python3.8
+
+WORKDIR $MAGMA_ROOT
+RUN mkdir $PYTHON_LIB/gen && \
+    for PROTO_SRC in orc8r lte feg; \
+    do \
+    python protos/gen_protos.py ${PROTO_SRC}/protos ${MAGMA_ROOT},orc8r/protos/prometheus ${MAGMA_ROOT} $PYTHON_LIB/gen && \
+    python protos/gen_prometheus_proto.py ${MAGMA_ROOT} $PYTHON_LIB/gen; \
+    done && \
+    echo "$PYTHON_LIB/gen" > $PYTHON_LIB/dist-packages/magma_gen.pth && \
+    find $PYTHON_LIB/gen -type d | tail -n +2 | sed '/__pycache__/d' | xargs -I % touch "%/__init__.py"
```

Comment:
Have you tried if this works without creating those `__init__.py`s?

Reply:
Yes, I tried. It didn't work for some of the files. I would have to dig deeper to understand why. But it might be worth it.

Reply:
Ok, I think it works as well. `module` in `from package_name import module` is found and the IDE forwards you to the code if you wish.  But it doesn't understand how to navigate to `package_name`. That might be because it is a namespace package (two possible paths are available and it doesn't know which one to pick) or may be even without the ambiguity it would be missing the __init__.py  to forward you to, not sure.

Reply:
An example from my VSCode:

<img width="392" alt="Screenshot 2022-02-03 at 18 28 20" src="https://user-images.githubusercontent.com/82914459/152505720-91432c6d-44c3-4efb-9b99-894c5790c3dd.png">